jetty-documentation/src/main/asciidoc/administration/startup/startup-base-vs-home.adoc

```diff
@@ -18,7 +18,7 @@
 === Managing Jetty Base and Jetty Home
 
 Instead of managing multiple Jetty implementations out of several different distribution locations, it is possible to maintain a separation between the binary installation of the standalone Jetty (known as `${jetty.home}`), and the customizations for your specific environment(s) (known as `${jetty.base}`).
-There should always only be one Jetty Home, but there can be multiple Jetty Base directories that references.
+There should always only be *one* Jetty Home, but there can be multiple Jetty Base directories that references.
```

Comment:
Only 1 jetty home will be used, but you can have more than 1 present on your file system.
Eg: Its especially useful to know that if your organization has a backout plan to test new versions of jetty, but allow for a quick fallback to older versions.